{{cookiecutter.repostory_name}}/envs/prod/docker-compose.yml

```diff
@@ -107,7 +107,7 @@ services:
   {% endif -%}
 
   nginx:
-    image: 'wojciechbancer/nginx-rt'
+    image: 'ghcr.io/reef-technologies/nginx-rt:master'
```

Comment:
this will explode for old clients if we ever make a non-forward compatbile change in the future!

Reply:
Action created should publish a version also based on tag. 
However i'm not able to apply tags to reef repo.

Reply:
ok lets do v1.0.0 then and point the image here to that.